kubernetes.md

```diff
@@ -0,0 +1,96 @@
+# Running cri-o on kubernetes cluster
+
+## Switching runtime from docker to cri-o
+
+In standard docker kubernetes cluster, kubelet is running on each node as systemd service and is taking care of communication between runtime and api service.
+It is reponsible for starting microservices pods (such as `kube-proxy`, `kubedns`, etc. - they can be different for various ways of deploying k8s) and user pods.
+Configuration of kubelet determines what runtime is used and in what way.
+
+Kubelet itself is executed in docker container (as we can see in `kubelet.service`), but, what is important, **it's not** a kubernetes pod (at least for now), 
```

Comment:
@rhatdan Yes, it is running in docker container but it's not visible for kubernetes as a pod. I wanted to highlight it, because some may be confused that we are using cri-o+runC for all the pods that are running on this node, but kubelet itself is running inside docker container.